charts/kueue/tests/kueueviz_test.yaml

```
@@ -0,0 +1,113 @@
+suite: test kueueviz deployment
+templates:
+  - kueueviz/backend-deployment.yaml
+  - kueueviz/frontend-deployment.yaml
+  - kueueviz/backend-ingress.yaml
+  - kueueviz/frontend-ingress.yaml
+tests:
+  - it: should set the pod priorityClassName when provided
```

```suggestion
  - it: should set the priorityClassName on the backend Deployment when provided.
```

Maybe like this? We have a similar test case for the frontend.